include/sdf/parser.hh

```diff
@@ -333,6 +333,17 @@ namespace sdf
   SDFORMAT_VISIBLE
   bool convertFile(const std::string &_filename, const std::string &_version,
                    const ParserConfig &_config, SDFPtr _sdf);
+  /// \brief Convert an SDF file to a specific SDF version.
```

Comment:
nit: add blank line above

Reply:
Added in https://github.com/gazebosim/sdformat/pull/1123/commits/fc741b23a57e481961db9352af92a41d8fe9d9f5.